cf-check/observables.c

```diff
@@ -0,0 +1,90 @@
+#include <platform.h>
+#include <observables.h>
+#include <logging.h>
+#include <definitions.h> // CF_BUFSIZE
+#include <known_dirs.h> // GetStateDir()
+#include <file_lib.h> // FILE_SEPARATOR
+#include <alloc.h> // xstrdup()
+#include <db_structs.h> // observable_strings
+
+void GetObservableNames(
+    const char *ts_key_path,
+    const char const** obnames)
+{
+    char buf[CF_BUFSIZE];
+    const char *filename;
+    const char const* temp[CF_OBSERVABLES] = {};
+    int i;
+
+    if (ts_key_path == NULL)
+    {
+        snprintf(buf, CF_BUFSIZE - 1, "%s%cts_key", GetStateDir(), FILE_SEPARATOR);
+//        fprintf(stderr, "No ts_key_path provided, using default path of '%s'", buf);
+        filename = buf;
+    }
+    else
+    {
+        filename = ts_key_path;
+    }
+
+    FILE *f = safe_fopen(filename, "r");
+    if (f == NULL)
+    {
+//        fprintf(stderr, "No ts_key path and no file at default path so using built-in observables information.");
```

Comment:
remove commented out code, how to provide feedback in this case where the dump should be JSON?
throw an error instead? In this case, no, it's just an FYI/info message.